src/app/movies/[id]/edit/page.tsx

```diff
@@ -0,0 +1,40 @@
+import { NextImage } from '@/components/NextImage';
+import { VerticalContainer } from '@/components/VerticalContainer';
+import prisma from '@/lib/prisma';
+import {MovieForm} from "@/components/MovieForm";
+
+export default async function MoviePage({
+  params: { id },
+}: {
+  params: { id: number };
+}) {
+  const movie = await prisma.movie.findUnique({
+    where: {
+      id: +id,
+    },
+  });
+
+  if (movie) {
+    return (
+      <VerticalContainer>
+        <div className='grid grid-cols-2 gap-8'>
+          <div>
+            <NextImage
```

Comment:
Height of image should adapt to height of form

Reply:
@neno i updated this but i would keep current solution. If we use same height than image aspect is lost and it would look worse in my case.